### DISCUSSION TITLE
How about adding a UPC to a listing?

### DISCUSSION THREAD

**Discussion by tahirkahnicloud** [etsy/open-api#1018] [Ideas]
For my use case this is very relevant. Would others find it useful if there was a UPC identifier on the listing?

**Comment by Gareth064** [comment]
Can you define a UPC? I haven't heard of this term before.

**Reply by tahirkahnicloud** [reply]
Universal Product Code. Here is a resource you can find more info on, https://en.wikipedia.org/wiki/Universal_Product_Code. Hope you are not kidding.

**Reply by Gareth064** [reply]
I wouldn't have heard of it because I don't see the need for barcodes for handmade or vintage items.

Sounds like the type of thing needed for mass produced, imported goods that you would find somewhere like Amazon. The type of stuff that handmade sellers are complaining to Etsy about all the time.

I'm curious what your use case is for it.

But to answer your question. No there is no field for UPC. The closest thing you have is a SKU.

**Comment by tahirkahnicloud** [comment]
I sometimes shop on Etsy and I have bought fitness related items on Etsy that had a barcode on the box. Since sellers can list items with barcodes on them having an optional UPC field would perfectly fit this use case, which happens to be mine too for the integration we are working on.

**Reply by Gareth064** [reply]
Yup, I get what your saying. But what I'm pointing out is that Etsy not meant to be the place to sell stuff like that. Unfortunately the platform has got too big for Etsy to police every listing to make sure it fits their idea of what is meant to be sold on Etsy.

If you want to formally request a new field be added then email developer@etsy.com with your request. The Etsy devs don't use this repo discussion to take requests or manage bugs etc.
